angular-workspace/projects/ni/nimble-angular/src/thirdparty/testing/matchers.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace jasmine {
```

Comment:
Is this file just created by us? Can you comment at the top if so and anything that needs to be done to update it?